src/services/user/adminUnblockUser.js

```diff
@@ -1,12 +1,14 @@
 const {successResponse, errorResponse} = require('../../utils');
 const {adminUnblockUserProcess} = require('../../processes/scoring/admin-block-unblock-user');
+const UsersFunction = require('../../databases/functions/users');
 
 const adminUnblockUser = async (req, res) => {
   try {
     const {userId} = req.body;
     if (!userId) {
       return errorResponse(res, 'userId is required', 400);
     }
+    await UsersFunction.userUnblockByAdmin(userId);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There is no error handling for the `adminUnblockUserProcess` function. If this function throws an error, it will not be caught and handled properly. Consider adding a try-catch block around this function call as well.
```diff
-    const updateResult = await adminUnblockUserProcess(userId);
+    let updateResult;
+    try {
+      updateResult = await adminUnblockUserProcess(userId);
+    } catch (error) {
+      return errorResponse(res, 'Failed to process unblock user by admin', 500);
+    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->